test/patches/ignored.txt

```diff
@@ -0,0 +1 @@
+This file is ignored, since we tell mysql-patcher to look for files with a particular filename.
```

Comment:
Nice, and it may be worth adding some more complex edge-cases here as well, e.g. `patch-ignored-foo-bar.sql`.